hieradata/type/staffvm.yaml

```diff
@@ -0,0 +1,3 @@
+ocf::auth::ulogin: [['%{owner}', 'ALL']]
+ocf::auth::usudo: ['%{owner}']
+ocf::auth::nopasswd: true
```

Comment:
Cool, these are perfect values to go in hiera in my opinion, they make it a lot more clear who is getting these settings and who isn't.